<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PixelInfoDialog</class>
 <widget class="QDialog" name="PixelInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixel Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leMousePos"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblRgb">
     <property name="text">
      <string>RGB #</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leRgb"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
